### PR TITLE
Refactor/update contract address

### PR DESCRIPTION
**Pull request by coreyar**
## Description

Adds contract addresses missing between the two network configs

### CODE REVIEW

File: networks/mainnet.json [hunk 1]
```diff
@@ -27,14 +28,29 @@
...
-    "XVSVaultProxy": "0x0000000000000000000000000000000000000000",
-    "XVSStore": "0x0000000000000000000000000000000000000000",
     "WBNB": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
-    "pancakeFactory": "0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73"
+    "pancakeFactory": "0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73",
+    "XVSVault": "0x6eF49b4e0772Fe78128F981d42D54172b55eCF9F",
+    "XVSVaultProxy": "0x051100480289e704d20e9DB4804837068f3f9204",
+    "XVSStore": "0x1e25CF968f12850003Db17E0Dba32108509C4359",
+    "SnapshotLens": "0x9802448Af040DA880Bf51bCbe4f2A4205ebC6d2C",
+    "VTreasury": "0x0000000000000000000000000000000000000000",
```
Comment (kkirka) [inline networks/mainnet.json:41]:
```suggestion
    "VTreasury": "0xF322942f644A996A617BD29c16bd7d231d9F35E9",
```

File: networks/mainnet.json [hunk 2]
```diff
@@ -27,14 +28,29 @@
...
+    "GovernorAlpha2": "0x388313BfEFEE8ddfeAD55b585F62812293Cf3A60",
+    "GovernorBravoDelegate": "0x0000000000000000000000000000000000000000",
```
Comment (kkirka) [inline networks/mainnet.json:43]:
```suggestion
    "GovernorBravoDelegate": "0x360ac19648efC29d2b7b70baC227C35e909272Fd",
```

File: networks/mainnet.json [hunk 3]
```diff
@@ -27,14 +28,29 @@
...
+    "GovernorBravoDelegator": "0x2d56dC077072B53571b8252008C60e945108c75a",
+    "Base200bps_Slope2000bps_Jump2000bps_Kink90": "0x0000000000000000000000000000000000000000",
```
Comment (kkirka) [inline networks/mainnet.json:45]: Do we need it? Interest rate models are different for different tokens and are deployed every time there's a recommendation from quants or gauntlet, I don't think we should track these

File: networks/mainnet.json [hunk 4]
```diff
@@ -27,14 +28,29 @@
...
+    "VRTVault": "0x0000000000000000000000000000000000000000",
```
Comment (kkirka) [inline networks/mainnet.json:46]:
```suggestion
    "VRTVault": "0xA3EEA5e491AD45caE30F6E0a315A275cc99EE147",
```

File: networks/mainnet.json [hunk 5]
```diff
@@ -27,14 +28,29 @@
...
+    "VRTVaultProxy": "0x98bF4786D72AAEF6c714425126Dd92f149e3F334",
+    "VRTConverter": "0xF0fCB229031A386C672e291cE183390c1a7705dA",
+    "VRTConverterProxy": "0x92572fB60f4874d37917C53599cAe5b085B9Facd",
+    "XVSVesting": "0x0000000000000000000000000000000000000000",
```
Comment (kkirka) [inline networks/mainnet.json:50]:
```suggestion
    "XVSVesting": "0xA9dD0536Ab0699D60c28CE369Ff366a1e621BFf1",
```

File: networks/mainnet.json [hunk 6]
```diff
@@ -27,14 +28,29 @@
...
+    "XVSVestingProxy": "0xb28Dec7C7Ac80f4D0B6a1B711c39e444cDE8B2cE",
+    "ComptrollerLens": "0x0000000000000000000000000000000000000000",
```
Comment (kkirka) [inline networks/mainnet.json:52]:
```suggestion
    "ComptrollerLens": "0x50F618A2EAb0fB55e87682BbFd89e38acb2735cD",
```

File: networks/mainnet.json [hunk 7]
```diff
@@ -27,14 +28,29 @@
...
+    "Liquidator": "0x0000000000000000000000000000000000000000"
```
Comment (kkirka) [inline networks/mainnet.json:53]: (This one is an openzeppelin proxy)

```suggestion
    "Liquidator": "0x0870793286aaDA55D39CE7f82fb2766e8004cF43"
```

File: networks/mainnet.json [hunk 8]
```diff
@@ -27,14 +28,29 @@
     "vETH": "0xf508fCD89b8bd15579dc79A6827cB4686A3592c8",
     "vLTC": "0x57A5297F2cB2c0AaC9D554660acd6D385Ab50c6B",
     "vXRP": "0xB248a295732e0225acd3337607cc01068e3b9c10",
+    "vTRX": "0xC5D3466aA484B040eE977073fcF337f2c00071c1",
```
Comment (kkirka) [inline networks/mainnet.json:31]: Do we need vTRXOLD as well?

**Review by kkirka — APPROVED** [review]

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**